### PR TITLE
119: As a user I want to have code component

### PR DESCRIPTION
**Pull request by Zoranais**
example

### CODE REVIEW

**Review by VladTkach — COMMENTED** [review]
Pleace add scroll if there ara a lot of lines
image
And also add line transfer id line is too long
image

can you center it?
image

And I think you need to rename code section or add some functionality to enter text because we need another one for scripts with ability to edit, but I'm not sure if we need ro edit ability in changes section.

File: frontend/src/app/shared/components/code/code.component.html [hunk 1]
```diff
@@ -3,17 +3,15 @@
         <h5 class="name">
             Solution
         </h5>
-        <img src="assets/full-size.svg" alt="full-size">
```
Comment (VladTkach) [inline frontend/src/app/shared/components/code/code.component.html]: According to design we have resize btn, but question is whether we need to add two sections for the code in one screen

Comment (ABertashDev) [reply frontend/src/app/shared/components/code/code.component.html]: But according to task - we have only one section and don't have resize button: [task](https://trello.com/c/LhLLhxmS/119-as-a-user-i-want-to-have-code-component)

**Comment by Zoranais** [comment]
example

**Review by VladTkach — COMMENTED** [review]
image
buttons at usual scale overlap other elements

**Comment by Zoranais** [comment]
> image buttons at usual scale overlap other elements

There was a problem with height of the tree component. Now it should be great.

**Review by VladTkach — COMMENTED** [review]
image
can you keep solution name on the top on scrolling?

**Review by MaxShelestiuk — APPROVED** [review]

**Review by VladTkach — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**